### ISSUE TITLE
cloud.action giving errors on 2016.11.1

### ISSUE THREAD

**Issue by UtahDave** [saltstack/salt#40278]
### Description of Issue/Question
on 2016.11.1 we're seeing this error:

```
[root@salt-1 ~]# salt salt-1 cloud.action start instance=test-minion01
salt-1: 
Passed invalid arguments to cloud.action: start() got multiple values for keyword argument 'call'

Execute a single action on the given provider/instance

CLI Example:

.. code-block:: bash

salt '*' cloud.action start instance=myinstance 
salt '*' cloud.action stop instance=myinstance 
salt '*' cloud.action show_image provider=my-ec2-config image=ami-1624987f

------------------------------------------- 
Summary 
------------------------------------------- 
# of minions targeted: 1 
# of minions returned: 1 
# of minions that did not return: 0 
# of minions with errors: 0 
-------------------------------------------
```

**Comment by UtahDave** [comment]
ZD 1257

**Comment by rallytime** [comment]
@UtahDave I have fixed this with #40572. This will be in the 2016.11.4 release.
